test/e2e/ingress/sanity.go

```diff
@@ -186,3 +186,49 @@ var _ = ginkgo.Describe("leader election", func() {
 		}
 	})
 })
+
+var _ = ginkgo.Describe("stream_routes disabled", func() {
+	opts := &scaffold.Options{
+		Name:                  "default",
+		Kubeconfig:            scaffold.GetKubeconfig(),
+		APISIXConfigPath:      "testdata/apisix-stream-disabled.yaml",
+		IngressAPISIXReplicas: 1,
+		HTTPBinServicePort:    80,
+		APISIXRouteVersion:    "apisix.apache.org/v2beta3",
+	}
+	s := scaffold.NewScaffold(opts)
+	ginkgo.It("/ip should return your ip", func() {
```

Comment:
What's the correlation between this case and "stream route disabled"?

Reply:
@tokers please see #856  
Previously, if this stream_route was disabled, it did not work properly

Reply:
OK. I suggest adding some comments to explain it as the case doesn't have any relations with the stream route.